src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -909,6 +909,33 @@ export class DBTCoreProjectIntegration
     }
   }
 
+  findPackageVersion(packageName: string) {
+    if (!this.packagesInstallPath || !packageName) {
+      return;
+    }
+
+    const dbtProjectYmlFilePath = path.join(
+      this.packagesInstallPath,
+      packageName,
+      "dbt_project.yml",
+    );
+    if (!existsSync(dbtProjectYmlFilePath)) {
+      return;
+    }
+    const fileContents = readFileSync(dbtProjectYmlFilePath, {
+      encoding: "utf-8",
+    });
+    if (!fileContents) {
+      return;
+    }
+    const parsedConfig = parse(fileContents, {
+      strict: false,
+      uniqueKeys: false,
+      maxAliasCount: -1,
+    });
+    return parsedConfig?.version;
```

Comment:
if version is not defined is also an error.

Reply:
fixed